frontend/src/scenes/experiments/MetricsView/new/MetricsTable.tsx

```diff
@@ -83,6 +83,14 @@ export function MetricsTable({
 
                         const isLoading = !result && !error && !!experiment.start_date
 
+                        // Hide details button/modal when ResultsBreakdown is shown for single primary funnel metrics
+                        const shouldHideDetails =
```

Comment:
Could we consolidate this with the logic that determines whether to show `ResultDetails` or not? It should be the same I guess? I think we want to include all metric types here also, not just funnels?